src/handlers/command-parser.ts

```diff
@@ -11,8 +11,18 @@ export class CommandParser {
     program
       .command("/wallet")
       .usage("<address>")
-      .argument("<address>", "Wallet address to query, e.g. 0x000000000000000000000000000000000000000", this._parseWalletAddress)
-      .action((address) => registerWallet(context, address))
+      .argument("[address]", "Wallet address to query, e.g. 0x000000000000000000000000000000000000000", this._parseWalletAddress)
+      .option("-u, --unset", "Unlink your wallet from your account")
```

Comment:
I realized that this double hyphen is really only geared towards programmers. We should support only /wallet unset syntax for everybody